sdks/python/apache_beam/runners/interactive/display/pcoll_visualization.py

```diff
@@ -181,6 +247,12 @@ def __init__(self, pcoll):
     self._overview_display_id = 'facets_overview_{}_{}'.format(
         self._cache_key, id(self))
     self._df_display_id = 'df_{}_{}'.format(self._cache_key, id(self))
+    # Whether the visualization should include window info.
```

Comment:
You can probably drop these comments. They are pretty obvious form variable names and use up to this point.